hypervisor/arch/x86/guest/vm.c

```diff
@@ -930,8 +935,14 @@ void prepare_vm(uint16_t vm_id, struct acrn_vm_config *vm_config)
 #ifdef CONFIG_SECURITY_VM_FIXUP
 	security_vm_fixup(vm_id);
 #endif
+	if (get_vmid_by_name(vm_config->name) == ACRN_INVALID_VMID) {
+		pr_err("Invalid VM name:%s", vm_config->name);
```

Comment:
```suggestion
		pr_err("Invalid VM name: %s", vm_config->name);
```

Reply:
Don't you want to insert `\n` at the end of the error message? Same question for all `pr_err` calls in this PR.

Reply:
no need to insert \n since it will add \n\r in do_logmsg called by pr_err